flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java

```diff
@@ -73,15 +73,15 @@ protected TableEnvironment getTableEnv() {
   }
 
   protected static TableResult exec(TableEnvironment env, String query, Object... args) {
-    return env.executeSql(String.format(query, args));
+    return env.executeSql(args.length > 0 ? String.format(query, args) : query);
```

Comment:
Why was this change needed?

Reply:
if the sql contains `%` , for example `LIKE  'a%'`, the `String.format` method will throws an excpetion, so I do the change.

Reply:
You can use `%%` in a format string to produce a literal `%`.

I think it would be better not to change this file, if it isn't necessary.

Reply:
ok,I will reverting it